contextCancelable.yml

```diff
@@ -10,6 +10,9 @@ rules:
           $_, $X := context.$CANCELABLE(...)
           ...
           $X()
+      - pattern-not-inside: |
+          $_, $X := context.$CANCELABLE(...)
+          t.Cleanup($X)
```

Comment:
Do we want a `...` here to allow people to have some source code lines *before* they call `t.Cleanup()` ?

Otherwise, LGTM.